bots/image-prune

```diff
@@ -144,13 +144,13 @@ def prune_images(force, dryrun, quiet=False, open_pull_requests=True, offline=Fa
         target = os.readlink(path)
         if not os.path.isabs(target):
             targets.add(os.path.join(testvm.IMAGES_DIR, target))
-            targets.add(os.path.join(DATA, target))
+            targets.add(os.path.join(testvm.get_images_data_dir(), target))
         else:
             targets.add(target)
 
     expiry_threshold = now - IMAGE_EXPIRE * 86400
-    for filename in os.listdir(DATA):
-        path = os.path.join(DATA, filename)
+    for filename in os.listdir(testvm.get_images_data_dir()):
+        path = os.path.join(testvm.get_images_data_dir(), filename)
```

Comment:
Maybe call the function once and save the value, as done in some other places?

Otherwise, obviously this is better than the current state (read: "not broken anymore").

Reply:
It's cheap enough, if not we can make it caching. Indeed this is rather urgent.